scripts/lib/build-utils.mjs

```diff
@@ -301,6 +318,8 @@ export default {
       tasks.push(() => this.buildNerdctlStub('linux'));
     }
     tasks.push(() => this.buildRdCtl(os.platform()));
+    tasks.push(() => this.buildVtunnel('windows'));
+    tasks.push(() => this.buildVtunnel('linux'));
```

Comment:
This should not happen on darwin / linux (put this in the same if block as nerdctl).